Mitmachen.md

```diff
@@ -18,12 +14,11 @@ Schreib uns einfach eine kurze E-Mail an <a href="mailto:mitmachen@athene-aachen
 ### Tätigkeit als Mentor_in
 Als Mentor_innen unterstützt du Lehrkräfte in Grund- und Sekundarschulen. Du begleitest den Unterricht und betreust bei Bedarf einzelne Kinder gezielt. Dafür musst du kein Mathe-Genie oder Sprachen-Ass sein. Etwas Zeit und Motivation ist alles, was du brauchst.
```

Comment:
Als Mentor_in unterstützt du Lehrkräfte in Grund- und Sekundarschulen. Du begleitest vormittags den Unterricht und betreust bei Bedarf einzelne Kinder gezielt oder hilfst nachmittags bei der Hausaufgabenbetreuung. Dafür musst du kein Mathe-Genie oder Sprachen-Ass sein. Etwas Zeit und Motivation ist alles, was du brauchst.